### ISSUE TITLE
Sent messages aren’t visible when using monochromatic theme

### ISSUE THREAD

**Issue by GreenBPartsUnknown** [christianselig/apollo-bugs#245]
When using monochromatic theme sent messages aren’t visible, both the bubble and text are white :P
![FECD48C2-485F-4552-ABB0-AC39271509CE](https://user-images.githubusercontent.com/54224054/63219124-9427ce00-c128-11e9-9edb-0dd73e9296d3.png)

**Comment by julian12981** [comment]
Same when using normal Dark-Mode

**Comment by GeeDuhb** [comment]
I don’t have the issue on dark mode, both regular and pure black. But it could also have to do with what theme you are using?

**Comment by GeeDuhb** [comment]
@julian12981 I don’t have the issue on dark mode, both regular and pure black. But it could also have to do with what theme you are using?

**Comment by GreenBPartsUnknown** [comment]
It’s definitely the monochrome theme, others work just fine :P

**Comment by julian12981** [comment]
@GeeDhub

It’s when I’m using no theme and dark mode

**Comment by GreenBPartsUnknown** [comment]
@Julian12981 so you’re getting this bug with the default theme?

**Comment by julian12981** [comment]
@GreenBPartsUnknown 

Yes, i attached a picture, the text when written is visible, but the text in the preview box isn’t.
![FAEEC889-E605-435D-9215-8EA97007B627](https://user-images.githubusercontent.com/54245806/63329841-fbb16980-c332-11e9-8c0e-321790cd1f36.png)

**Comment by GreenBPartsUnknown** [comment]
@Julian12981 Just tried composing a message and had your exact same problem. Tested it both with default and Spooky Pumpkin
